spec/models/maintenance_stats/github/release_stats_spec.rb

```diff
@@ -31,18 +31,18 @@
         
         # check values against the VCR cassette data
         expect(results[:last_week_releases]).to eql 1
-        expect(results[:last_month_releases]).to eql 2
-        expect(results[:last_two_month_releases]).to eql 2
-        expect(results[:last_year_releases]).to eql 2
+        expect(results[:last_month_releases]).to eql 1
+        expect(results[:last_two_month_releases]).to eql 1
+        expect(results[:last_year_releases]).to eql 5
 
-        expect(results[:last_release_date]).to eql "2019-01-18T21:32:44Z"
+        expect(results[:last_release_date]).to eql "2019-01-05T15:47:46Z"
     end
   end
   
   context "repository with no commits" do
     let(:repository) { create(:repository, full_name: 'buddhamagnet/heidigoodchild') }
     let(:query_results) do
-        VCR.use_cassette('github/empty_repository', :match_requests_on => [:body]) do
+        VCR.use_cassette('github/empty_repository', :match_requests_on => [:method, :uri, :body, :query]) do
```

Comment:
Is there anything special that needs to be done to re-record these cassettes? If so, now might be good time to document it for posterity.

Reply:
Where would be a good spot for information like that?

Reply:
In the past I've just plopped it in comments near the VCR calls